### ISSUE TITLE
Convenience improvement on level switching

### ISSUE THREAD

**Issue by CharlesChou73** [enriquein/AddAmountOfRecipePer1min#1]
Copied from last issue:
<br>

Switching the displayed speed among diffirent structure level is a good idea, but it's a bit of annoying that every time I want to check productivity in mk3, I have to press Alt. Usually speaking, mk3 is the most used level.

So here are some methods to improve it:

Easiest, set mk3 as the default displaying level.

A little bit harder, add a config file to set the default level.

More harder but convenient, record the level that user checked last time, set it as the default level.

Hardest but perfect, add a combo box somewhere, maybe in the replicator panel, allowing user to select the default level.

<br>
Thanks again.

**Comment by enriquein** [comment]
I updated the mod to include separate configuration for assembler and smelter speeds. To access the configuration, run the game at least once after updating the mod, close the game, and then use r2modman's config editor to change the values. 

I decided to implement this as a configuration instead of an in-game setting because I'm not comfortable doing UI work yet. I think your idea of having an in-game list with the option would be the ideal solution, but for now this will have to do.
